models/trainer/gan_trainer.py

```diff
@@ -50,6 +51,7 @@ def __init__(self,
             - modelLabel (string): name of the model
             - config (dictionary): configuration dictionnary.
             for all the possible options
+            - max_time (int): max number of seconds for training (0 = infinity).
```

Comment:
NIT: working with second is not very practical, hours looks like a better unit to me.